src/main/java/org/broadinstitute/consent/http/db/mapper/InstitutionMapper.java

```diff
@@ -36,18 +38,97 @@ public Institution map(ResultSet resultSet, StatementContext statementContext) t
       institution.setItDirectorEmail(resultSet.getString("it_director_email"));
     }
     if (hasColumn(resultSet, "create_user")) {
-      institution.setCreateUser(resultSet.getInt("create_user"));
+      institution.setCreateUserId(resultSet.getInt("create_user"));
     }
     if (hasColumn(resultSet, "create_date")) {
       institution.setCreateDate(resultSet.getDate("create_date"));
     } 
     if (hasColumn(resultSet, "update_user")) {
-      institution.setUpdateUser(resultSet.getInt("update_user"));
+      institution.setUpdateUserId(resultSet.getInt("update_user"));
     }
     if (hasColumn(resultSet, "update_date")) {
       institution.setUpdateDate(resultSet.getDate("update_date"));
     }
+
+    User createUser = new User();
+
+    if (hasColumn(resultSet, "dacUserId")) {
+      createUser.setDacUserId(resultSet.getInt("dacUserId"));
+    } 
+    if (hasColumn(resultSet, "email")) {
+      createUser.setEmail(resultSet.getString("email"));
+    }
+    if (hasColumn(resultSet, "displayName")) {
+      createUser.setDisplayName(resultSet.getString("displayName"));
+    }
+    if (hasColumn(resultSet, "createDate")) {
+      createUser.setCreateDate(resultSet.getDate("createDate"));
+    }
+    if (hasColumn(resultSet, "additional_email")) {
+      createUser.setAdditionalEmail(resultSet.getString("additional_email"));
+    }
+    if (hasColumn(resultSet, "email_preference")) {
+      createUser.setEmailPreference(resultSet.getBoolean("email_preference"));
+    }
+    if (hasColumn(resultSet, "status")) {
+      createUser.setStatus(getCreateStatus(resultSet));
+    }
+    if (hasColumn(resultSet, "rationale")) {
+      createUser.setRationale(resultSet.getString("rationale"));
+    }
+    //user model does not currently have an institutionId field or methods
+    // if (hasColumn(resultSet, "institute")) {
+    //   user.setInstitutionId(resultSet.getInt("update_user"));
+    // }
```

Comment:
Currently the dacuser table has institution_id as a column, but the model does not have a field for this property. If this is something we want to implement I will file a ticket otherwise I will just delete these commented lines.

Reply:
prefer to remove the comments until we need to implement it.